opendm/config.py

```diff
@@ -17,28 +31,34 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def config():
     parser.add_argument('--images', '-i',
-                        metavar='<string>',
+                        metavar='<path>',
                         help='Path to input images'),
 
     parser.add_argument('--project-path',
-                        metavar='<string>',
-                        help='Path to the project to process')
+                        metavar='<path>',
+                        help='Path to the project folder',
+                        default=defaultSettings['project_path'])
+
+    parser.add_argument('name',
+                        metavar='<project name>',
+                        type=alphanumeric_string,
+                        help='Name of Project (i.e subdirectory of projects folder)')
 
     parser.add_argument('--resize-to',  # currently doesn't support 'orig'
                         metavar='<integer>',
-                        default=2400,
+                        default=defaultSettings['resize_to'],
```

Comment:
I simply hardcoded them in the yaml file rather than here because that was simplest place to write the changes. Now they can be updated by the user in that interface. I think it would be confusing if the defaults were in both, esp. if they changed in one or the other, not knowing which takes precedence.